aiida/common/log.py

```diff
@@ -201,3 +202,15 @@ def configure_logging(with_orm=False, daemon=False, daemon_log_file=None):
         config['loggers']['aiida']['handlers'].append(handler_dblogger)
 
     dictConfig(config)
+
+
+@contextmanager
+def overide_log_level(level=None):
```

Comment:
overide -> override

Reply:
Ah damn.. I was so close to double checking that and then just went ahead. :D